app/components/focusing-outlet.js

```diff
@@ -57,7 +59,11 @@ let FocusingOutlet = Ember.Component.extend({
       let pivotHandler = owner.get('_stashedHandlerInfos.pivotHandler.handler.routeName');
 
       let outletName = this.get('outletName');
-      let currentRoute = outletState[outletName].render.name;
+
+      let currentRoute = get(outletState, `${outletName}.render.name`);
+      if (isEmpty(currentRoute)) {
```

Comment:
Since we know that it will either be a string or `undefined` [`isEmpty`](https://github.com/emberjs/ember.js/blob/v2.5.0/packages/ember-metal/lib/is_empty.js) is overkill.